Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -7395,7 +7395,10 @@ bool WebPage::shouldDispatchUpdateAfterFocusingElement(const Element& element) c
 
 static bool isTextFormControlOrEditableContent(const WebCore::Element& element)
```

Comment:
Consider renaming this since that's not what the function returns anymore?

Reply:
left this function as-is and added a new one.